providers/common/include/prov/providercommonerr.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2020-2020 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Undo this please

Reply:
Fixed.  I'm not at all sure how this got here.